test/unit/libp2p_port_test.exs

```diff
@@ -86,37 +87,31 @@ defmodule Unit.Libp2pPortTest do
     start_port(:publisher)
     start_port(:gossiper, listen_addr: gossiper_addr)
 
+    # The message sent is the pid
```

Comment:
Let's explain why we do this (we use the message in this module's handler to decode the pid and send a notification)